util/reggen/reg_block.py

```diff
@@ -82,13 +83,18 @@ def build_blocks(block: 'RegBlock',
         key/value pairs start empty RegBlocks. The return value is a dictionary
         mapping the keys we saw to their respective RegBlocks.
 
+        The flag is_alias determines whether this block contains alias register
```

Comment:
I think this should be "these blocks contain"?

Reply:
Fixed